lib/os/thread_posix.cpp

```diff
@@ -44,10 +50,18 @@
 
 namespace pstore {
     namespace threads {
+#    if defined(PSTORE_PTHREAD_SETNAME_NP_1_ARG) || defined(PSTORE_PTHREAD_SETNAME_NP_2_ARGS) ||   \
+        defined(PSTORE_PTHREAD_SETNAME_NP_3_ARGS) || defined(PSTORE_PTHREAD_SET_NAME_NP)
+#        define PTHREAD_SETNAME_NP
```

Comment:
I would define as 1 rather than leaving empty. This follows the convention used in config.hpp and avoids a potential error if someone checks it with #if rather than #ifdef.

Reply:
Fixed in the commit https://github.com/SNSystems/pstore/pull/93/commits/ed5cf97fa57af25655999699db16ec75c2c7fe96.